src/main/config/config.c

```diff
@@ -132,6 +138,11 @@ static void loadPidProfile(void)
     currentPidProfile = pidProfilesMutable(systemConfig()->pidProfileIndex);
 }
 
+static void loadCurrentControlRateProfile(void)
```

Comment:
You don't need to do this here - this is already done in `loadControlRateProfile()`.

Reply:
ok